src/comment.ts

```diff
@@ -14,12 +14,15 @@ export async function findPreviousComment(
   issue_number: number,
   header: string
 ): Promise<{body?: string; id: number} | undefined> {
+  const {viewer} = await octokit.graphql("query { viewer { login } }")
```

Comment:
Had to use the GraphQL API here, octokit.rest.users.getAuthenticated() fails using the default GITHUB_TOKEN (permissions issue?)